bucket/jabref.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "5.1",
+    "description": "Free Reference Manager",
+    "homepage": "https://www.jabref.org",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/JabRef/jabref/blob/master/LICENSE.md"
+    },
+    "url": "https://www.fosshub.com/JabRef.html/JabRef-5.1-portable_windows.zip",
+    "hash": "1a0aaa60c7424a356e3e288cc27218ddd5fe8e85ae295d87a561960404318666",
```

Comment:
There is not a single reason to prefer fosshub over github

Reply:
Updated. See commit "Update jabref.json"